### ISSUE TITLE
TREX: Can't find nonce with device [ID=0, GPU #0], cuda exception in [StreamContext<struct search_results,struct Ethash::KernelLaunchTag>::synchronize, 51], unknown error

### ISSUE THREAD

**Issue by MASKFER38** [trexminer/T-Rex#376]
Hello!
How you doing?

In this image I show you the error I get while trying to run MSI setup with undervolt and 1200 mhz on a RTX 3060 Twin Edge OC 12GB
[https://ibb.co/bQvZv2X](url)
As you can see one hour and half passed and the error came out dropping the MH/s to 6+-
So I got back to -500 Clock +1000 Memory without undervolt and got 47+- MH/s

Any thoughts?

**Comment by c3rtified** [comment]
I am having the same issue, were you able to find a solution to this?

**Comment by MASKFER38** [comment]
> I am having the same issue, were you able to find a solution to this?

No. Or I sucks at google or Nividia must drop an official driver to unlock the hashrate 100% with no issues.

**Comment by StealthSignatures** [comment]
I'm on Windows. Tried everything, it must be GPU hardware related(i checked risers, cables, etc). My GPU used to get a highier mem clock, but after a repaste my GPU triggered the problem few minutes after loading DAG, every single time. Reducing mem clock was the only way to make it work properly, but less hashrate(-5%).
As an extra detail, i need to reboot to make changes on both clocks or it triggers the error after some seconds.

**Comment by MASKFER38** [comment]
Great! tyvm!
I ended up with:
-100
+1099
70TP
65Fan

48.03Mh/s It is stable 99%.
